obspy/clients/earthworm/__init__.py

```diff
@@ -18,26 +18,26 @@
 
     >>> from obspy.clients.earthworm import Client
     >>> client = Client("pubavo1.wr.usgs.gov", 16022)
-    >>> response = client.get_availability('AV', 'KCG', channel='EHE')
+    >>> response = client.get_availability('AV', 'AKV', channel='BHE')
     >>> print(response)  # doctest: +SKIP
     [('AV',
-      'ACH',
+      'AKV',
       '--',
       'BHE',
-      UTCDateTime(2020, 4, 30, 12, 2, 27, 473000),
-      UTCDateTime(2020, 6, 29, 12, 2, 16, 899000)]
+      2021-12-30T14:36:15.761000Z,
+      2022-02-28T14:35:57.199000Z)]
```

Comment:
this is skipped so it doesnt matter too much anyway, but if I do that `print()` I *do* get `UTCDateTime(...)` and not some plain string

Reply:
Mee, too, in python and ipython. Maybe I used it inside spyder?